Source_Code/TM_UnitTests/TeamMentor.UnitTests/CoreLib/Schemas/Test_Schemas_Validation.cs

```diff
@@ -75,15 +75,17 @@ public void Validation_NewUser_LargeDataInFields()
                              "mappings : {0}".format(result.Value.toString()));
         }
         
-        [Test]
+        [Test,Ignore]

```

Comment:
I still think that we should keep this test (unless the tests below are doing the exact check)